### DISCUSSION TITLE
Migrating to null safety, how do I migrate generated files?

### DISCUSSION THREAD

**Discussion by cervonwong** [simolus3/drift#1223] [Q&A] [answered]
Do I accept the changes made by the migrator tool (there are like 800+ changes for my .g.dart file).

Or do I run build_runner again? Do I just run the normal command?

**Comment by simolus3** [comment, marked as answer]
You can temporarily accept the changes by the migrator tool, but I definitely recommend re-running the builder (perhaps with `--delete-conflicting-outputs` if you made changes to generated files). Moor will generate null-safe code when the input file is migrated to null safety as well.

**Reply by cervonwong** [reply]
Thanks!
